region/brinstar/red/Alpha Power Bomb Room.json

```diff
@@ -105,6 +105,94 @@
       },
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "Leave Spinning (Space Jump)",
+      "requires": [
+        {"obstaclesCleared": ["B"]},
+        {"or": [
+          {"obstaclesCleared": ["A"]},
+          "canTrickyJump"
+        ]},
+        "SpaceJump"
+      ],
+      "exitCondition": {
+        "leaveSpinning": {
+          "remoteRunway": {
+            "length": 11,
+            "openEnd": 1
+          }
+        }
+      }
+    },
+    {
+      "link": [1, 1],
+      "name": "Leave With Mockball (Space Jump)",
+      "requires": [
+        {"obstaclesCleared": ["B"]},
+        {"or": [
+          {"obstaclesCleared": ["A"]},
+          "canTrickyJump"
+        ]},
+        "SpaceJump"
+      ],
+      "exitCondition": {
+        "leaveWithMockball": {
+          "remoteRunway": {
+            "length": 11,
+            "openEnd": 1
+          },
+          "landingRunway": {
+            "length": 4,
+            "openEnd": 1
+          }
+        }
+      }
+    },
+    {
+      "link": [1, 1],
+      "name": "Leave With Spring Ball Bounce (Space Jump)",
```

Comment:
Looks good, added strats for controlled springball bounce.

Reply:
I've been avoiding putting in uncontrolled bounces where you can't control the position of the bounce, especially if you can't bounce right at the door, since it would be hard to know if it will match properly with what is needed in the next room. We may be able to refine it later; I figure it may become more clear once more of the applications are written